### PR TITLE
Note about username format

### PR DESCRIPTION
**Pull request by ManikaDhiman**

### CODE REVIEW

**Comment by Court72** [comment]
@sethmanheim

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by sethmanheim** [comment]
Looks good, thank you! #sign-off

File: azure-stack/hci/deploy/sdn-wizard.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ SDN Network Controller deployment is a functionality of the SDN Infrastructure e
         1. Specify the default gateway.
         1. Specify one or more DNS servers. Click **Add** to add additional DNS servers.
 1. Under **Credentials**, enter the username and password used to join the Network Controller VMs to the cluster domain.
+    > [NOTE]
```
Comment (Court72) [inline azure-stack/hci/deploy/sdn-wizard.md:84]:
```suggestion
    > [!NOTE]
```

File: azure-stack/hci/deploy/sdn-wizard.md [hunk 2]
```diff
@@ -145,6 +147,8 @@ SDN SLB deployment is a functionality of the SDN Infrastructure extension in Win
         1. Specify one or more DNS servers. Click **Add** to add additional DNS servers.
     
 1. Under **Credentials**, enter the username and password that you used to join the Software Load Balancer VMs to the cluster domain.
+    > [NOTE]
```
Comment (Court72) [inline azure-stack/hci/deploy/sdn-wizard.md:150]:
```suggestion
    > [!NOTE]
```

File: azure-stack/hci/deploy/sdn-wizard.md [hunk 3]
```diff
@@ -184,6 +188,8 @@ SDN Gateway deployment is a functionality of the SDN Infrastructure extension in
         1. Specify one or more DNS servers. Click **Add** to add additional DNS servers.
         
 1. Under **Credentials**, enter the username and password used to join the Gateway VMs to the cluster domain.
+    > [NOTE]
```
Comment (Court72) [inline azure-stack/hci/deploy/sdn-wizard.md:191]:
```suggestion
    > [!NOTE]
```

**Pull request merged**